JavaScript/Task_Day1/Printing_Array_Element.js

```diff
@@ -0,0 +1,6 @@
+//https://www.codewars.com/kata/56e2f59fb2ed128081001328/train/javascript
+//Printing Array Element
+
+function printArray(array) {
+  return array.join(",");
```

Comment:
Array method join, by default take comma(,) as the separator.
No need to specify a separator if you want to separate them by comma.

array.join();

Look into the document link below to learn more about array.join():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
Okay.